### PR TITLE
Add default values to config

### PR DESCRIPTION
**Pull request by Schlauer-Hax**
# Proposed Changes

I would like to add the default values of the config options that are available. It allows for more easier configuration of the add-on as you see instantly what is enabled and disabled. If there are no default values all values are turned off on the configuration page which does not reflect the actual value used by the add-on. This could lead to confusion, why an option is turned on, even if the user thinks it is turned off.

## Related Issues

No related Issues.

## Summary by CodeRabbit

- **New Features**
	- Introduced new configuration options for enhanced network settings, including DNS and route acceptance.
	- Added options for advertising exit nodes and specific route features, providing users greater control over their network setup.
	- Enhanced logging capabilities with adjustable log levels.
	- Expanded configurability with options for stateful filtering and userspace networking.
	- Updated schema to support new boolean options for improved configuration management.

### CODE REVIEW

File: tailscale/config.yaml [hunk 1]
```diff
@@ -25,6 +25,22 @@ devices:
   - /dev/net/tun
 map:
   - share:rw
+options:
+  accept_dns: true
+  accept_routes: true
+  advertise_exit_node: true
+  advertise_connector: true
+  advertise_routes:
+    - "match(^(((25[0-5]|(2[0-4]|1\\d|[1-9]?)\\d)\\.){3}(25[0-5]|(2[0-4]|1\\d|[1-9]?)\\d)\\/(3[0-2]|[12]?\\d)|[a-fA-F\\d.:]+:[a-fA-F\\d.:]+\\/(12[0-8]|(1[01]|[1-9]?)\\d))$)?"
```
[the remark this answers is not in the document]
Comment (Schlauer-Hax) [reply tailscale/config.yaml:34]: true, i overlooked that, thanks

**Comment by lmagyar** [comment]
I absolutely agree, that the curent UI is broken and misleading, unused options aren't grayed out but shown as false/empty. That's why I always use the yaml config view as a workaround.

Though I think it would be better to fix the UI. As I see, there are plans to refactor the options/schema structure (https://github.com/home-assistant/supervisor/issues/2640), so I think there won't be any fix for the current structure before that refactor happens. :/

**Comment by Schlauer-Hax** [comment]
I didn't know there was a change proposed to change the spec, sounds good. Until then we could add the default values so new users that don't know how to use the yaml config view aren't confused.

**Comment by frenck** [comment]
This is an issue with HA not the add-on; I am against solving UX problems in the backend.

Nevertheless; thanks for being willing to contribute 👍 

../Frenck

**Pull request closed without merging**